src/components/Footer/Footer.js

```diff
@@ -1,6 +1,18 @@
 import React from 'react'
+import { categories } from '../../backend/db/categories';
+import { accountLinks } from '../../backend/db/accountLinks';
+import { contactUs } from '../../backend/db/contactUs';
```

Comment:
Dont import from backend, make a network call to get it, if it is a static data so store it in frontend itself in a folder maybe name as data or statticData